src/Microsoft.DotNet.Interactive.Formatting/PocketViewTags.cs

```diff
@@ -85,5 +87,11 @@ public static class PocketViewTags
         public static dynamic u => _.u;
         public static dynamic ul => _.ul;
         public static dynamic video => _.video;
+
+        // This can be used to indicates points
+        // where we are embedding an arbitrary value into the HTML content
+        // and we are explicitly indicating that the value can be either IHtmlContent
+        // or will otherwise be formatted as plaintext.
+        public static object arbitrary(object value) => value;
```

Comment:
If I understand this correctly, it's intended to defer formatting? 

We also have the `HTML` method which wraps the object in an `IHtmlContent`, which is more explicit about the intent in the HTML case.

Reply:
Well, the intent is to mark the places we end up coming back around to either IHtmlContent or plain text formatting here https://github.com/dotnet/interactive/blob/master/src/Microsoft.DotNet.Interactive.Formatting/PocketView.cs#L213

Initially I chose the name `arbitraryIHtmlContentOrElseUsePlainTextFormatting` since it at least explains what's actually going to happen to this content :))

Reply:
Happy to change this however you see fit.  The use of `dynamic` and `object` makes it hard to mark up these recursive injection places (= deferred formatting I guess) except via a discpline, and I find them quite important for understanding what's going on.

Reply:
By deferred I guess I mean that by calling `arbitrary`, we're deferring the decision back to the formatter, but yes, it's not deferred by very long. 

Would `FormatToHtml` capture the intent?

Reply:
> Would FormatToHtml capture the intent?

Well, the current situation is to very often format as plaintext :) So it's kind of the opposite to `FormatToHtml` at the moment

Reply:
I'll leave it as `arbitrary` with tthe comment for now and we can resolve later

Reply:
To clarify, my understanding here is that we want `IHtmlContent` back (including if the value was generated by the `PlainTextFormatter` and then HTML-encoded). The current code doesn't know how to do that so it defers the decision back to the infrastructure. The result is plain text today but might eventually include nestable HTML.

Is this accurate? If so, is it useful to change `arbitrary` to something like:

```csharp
public IHtmlContent YouFigureItOut() => /*...*/
```